app/views/items/_items_table.html.erb

```diff
@@ -19,6 +20,7 @@
         <td><%= item_status_label(item) %></td>
         <td><%= item_condition_label(item) %></td>
         <td><%= item.location.name if item.location %></td>
+        <td><%= item.inspections.first.inspection_date %></td>
```

Comment:
I'm not requiring it, but what do you think of having an `@item.last_inspection_date` to incapsulate this in the model ?